internal/service/api/handlers/get_user_roles_map.go

```diff
@@ -0,0 +1,41 @@
+package handlers
+
+import (
+	"net/http"
+
+	"github.com/acs-dl/slack-module-svc/internal/data"
+	"github.com/acs-dl/slack-module-svc/resources"
+	"gitlab.com/distributed_lab/ape"
+)
+
+func GetUserRolesMap(w http.ResponseWriter, r *http.Request) {
+	result := newModuleRolesResponse()
+
+	result.Data.Attributes["super_admin"] = data.Roles[data.Owner]
+	result.Data.Attributes["admin"] = data.Roles[data.Admin]
+	result.Data.Attributes["user"] = data.Roles[data.Member]
+
+	ape.Render(w, result)
+}
+
+func newModuleRolesResponse() ModuleRolesResponse {
```

Comment:
there is `models` folder in api folder for such functions that converts objects from one to another